src/packages/core/efuns_main.cc

```diff
@@ -351,12 +351,22 @@ void f_capitalize(void) {
 #endif
 
 #ifdef F_CHILDREN
+//TODO:fix this. I basically moved code from the old children into this function but it should probably
```

Comment:
The documentation here need to state clear what exactly to fix. if the "fix" is not really clear yet you should simply remove it. Otherwise this will litter the codebase with no one know what exactly to do.